### ISSUE TITLE
Incorrect Build Action on generated ViewModels

### ISSUE THREAD

**Issue by lachlanwgordon** [mfractor/mfractor-feedback#102]
Hi Matt,
When I generate a View/ViewModel the ViewModel has the Build Action "None" when it should be "Compile". I found this when trying to add the view model as the binding context on the view.
See Screenshots below.

My settings for generating the view and the view model
![screen shot 2019-02-22 at 1 32 26 pm](https://user-images.githubusercontent.com/29908924/53216311-41ed7f00-36a8-11e9-9a47-66b882ce4106.png)

The default build action of the view model
![screen shot 2019-02-22 at 1 32 47 pm](https://user-images.githubusercontent.com/29908924/53216344-5893d600-36a8-11e9-996e-39c48838f765.png)

The error I get when trying to use the view model with the wrong build action
![screen shot 2019-02-22 at 1 45 08 pm](https://user-images.githubusercontent.com/29908924/53216365-69dce280-36a8-11e9-9860-3749c8d4dc99.png)

The generated view model
![screen shot 2019-02-22 at 1 45 20 pm](https://user-images.githubusercontent.com/29908924/53216380-78c39500-36a8-11e9-8fc7-4ca05657c006.png)

Let me know if you need any more info or help testing.

Cheers,
Lachlan

##Installation Information

```
--Platform Information--
Operating System: Mac
OS Version: 10.14.3
--MFractor Information--
Product Name: MFractor
Product Version: 3.7.8
Product Variant: Visual Studio For Mac
Product SKU: MFRACTOR-VS-MAC
Commit SHA: 7383209af583722ca8fa90c447536c939a932e9b
Revision: 3075
Branch: master
Date: 17/02/2019 9:06:32 PM (UTC)
Agent: Local
--IDE Information--
Mono Version: 5.18.0.248 (2018-08/a4956c837e1)
Runtime Version: 7.7.4.1
IDE Version: 7.7.4.1
IDE Product is: Visual Studio
Installed addins:
	'Brute Clean', 'MFractor', 'Color Helper Addin', '.Net Core Debugger', 'Razor Language Services', 'Web Editors', 'Web Project Support', 'ASP.NET Core Support', 'MonoDevelop Assembly Browser', 'Makefile generation', 'Azure Functions development', 'CSharp Language Binding', 'ChangeLog Add-in', 'ConnectedServices', 'Debugger support for MonoDevelop', 'Mono Soft Debugger Support', 'Mono Soft Debugger Support for ASP.NET', 'VsCode Debug Protocol support for MonoDevelop', 'Deployment Services Core', 'Deployment Services for Linux', 'Visual Designer Support', 'DocFood', 'Container Tools', '.NET Core Support', 'F# Language Binding', 'Gettext Translations Support', 'GTK# Visual Designer', 'MonoDevelop Hex Editor', 'ILAsm Language Binding', 'iOS development', 'Mac Development', 'MonoDevelop Mac Platform Support', 'Android development', 'IDE Extensions for developers of the Mono framework', 'Xamarin.Mac development', 'NuGet Package Management', 'NuGet Packaging', 'Performance Diagnostics of IDE', 'Refactoring Support', 'Regex Toolkit', 'MonoDevelop Source Editor', 'Test Runner', 'Text Templating', 'Unit Testing core support', 'NUnit support', 'Visual Studio for Mac Tools for Unity', 'VB.NET Language Binding', 'Version Control Support', 'Git support', 'Subversion core engine', 'Subversion support', 'Project Web References', 'Xamarin Ide Extensions', 'Xamarin Ide Extensions for Mac', 'XML Editor', 'Android Designer', 'Azure Mobile App Service', 'Azure Support', 'Xamarin Designer', 'Forms Project Support', 'Forms Previewer', 'Xamarin Ide Identity', 'Xamarin Ide Insights', 'Xamarin Inspector Support', 'Xamarin Live Player', 'Xamarin Test Cloud', 'Xamarin Test Cloud - NUnit', 'iOS Designer'

```

**Comment by matthewrdev** [comment]
Thanks for reporting this Lachlan. This has been fixed and will be available in the next release (3.7.9).

**Comment by matthewrdev** [comment]
Fixed and available in 3.7.9.
